analyzers/src/SonarAnalyzer.Common/Rules/StringLiteralShouldNotBeDuplicatedBase.cs

```diff
@@ -54,7 +54,7 @@ public abstract class StringLiteralShouldNotBeDuplicatedBase<TSyntaxKind, TLiter
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(rule);
 
         protected StringLiteralShouldNotBeDuplicatedBase() =>
-            rule = DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, Language.RspecResources, false);
+            rule = Language.CreateDescriptor(DiagnosticId, MessageFormat, isEnabledByDefault: false);
```

Comment:
Adding `isEnabledByDefault:` explicitly, like in every other case (I don't like it, it's for consistency)